src/components/smartBanner/SmartBanner.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import Config from '../../config/config'
+
+import '../../../node_modules/smartbanner.js/dist/smartbanner.min.js'
+
+import '../../../node_modules/smartbanner.js/dist/smartbanner.min.css'
+
+const SmartBannerMain = () => {
+  return (
+    <>
+      <meta name="smartbanner:title" content="GoodDollar - Claim crypto UBI" />
```

Comment:
this should be rendered to the `<head>`

1. include this component in src/index.js
2. add an additional ReactDOM.render call **before** rendering the main app

```javascript
ReactDOM.render(
  <SmartBanner />,
 document.head,
)
```